src/Benchmarks/Controllers/DbControllerBase.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Benchmarks.Data;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Benchmarks.Controllers
+{
+    [Route("mvc")]
+    public abstract class DbControllerBase : Controller
+    {
+        private RawDb _rawDb;
+        private DapperDb _dapperDb;
+        private EfDb _efDb;
+
+        protected RawDb RawDb
+        {
+            get
+            {
+                if (_rawDb == null)
```

Comment:
Alternatively, it's fine to leave this alone for now, and we can use another PR to make code cleanup improvements.  For example, I think `RawDb.cs` can be refactored to remove duplicate code.